nncf/experimental/openvino_native/graph/transformations/command_creation.py

```diff
@@ -16,22 +16,37 @@
 from nncf.common.graph.transformations.commands import TargetType
 from nncf.common.graph.graph import NNCFGraph
 from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.transformations.command_creation import CommandCreator
 from nncf.experimental.openvino_native.graph.transformations.commands import OVTargetPoint
 from nncf.experimental.openvino_native.graph.transformations.commands import OVBiasCorrectionCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVFQNodeRemovingCommand
+from nncf.experimental.openvino_native.graph.transformations.commands import OVWeightUpdateCommand
 
 
-def create_bias_correction_command(node: NNCFNode,
-                                   bias_value: np.ndarray,
-                                   nncf_graph: NNCFGraph) -> OVBiasCorrectionCommand:
+class OVCommandCreator(CommandCreator):
```

Comment:
Why the `ONNXCommandCreator` wasn't added?

Reply:
This is out of the scope of this PR. As we discussed with @alexsu52 this class will be added in the next PRs.